boggle.js

```diff
@@ -7,25 +9,23 @@ function newBoggleBoard() {
   ];
 }
 
-/**
- * Prints out a boggle board.
- */
 function printBoggleBoard(boggleBoard) {
   for (let row of boggleBoard) {
     console.log(row.join(' '));
   }
 }
 
-/**
- * Shake a boggle board and fill it with letters.
- */
-function shake(boggleBoard) {
-  // This is your job. :)
-  return boggleBoard;
-}
 
+function shake(board) {
+ for (a=0; a<=15; a++){
```

Comment:
Use ESLint! `a` is a global variable here.

Also, no reason to hard code `15` here.

```javascript
for (let a = 0; a < board.length; a++) {
  // ...
}
```